objects/relation.py

```diff
@@ -0,0 +1,39 @@
+from .helpers import value_to_variable
+from typing import List, Union
+
+class Relation:
+    def __init__(self, relation: str, values: List[str]=[]):
+        self.relation = relation
+        self.is_conditional = '|' in relation
+        self.is_joint = not self.is_conditional
```

Comment:
Will this works for relations that are joint and conditional? Not sure if we will have them in this assignment but I believe they are possible, such as `x , y | z`. In other words, I think we can have a joint and conditional, as you have in the tests below. I do not think they are mutually exclusive!